docs/ui/components/CommandMenu/utils.ts

```diff
@@ -99,6 +99,17 @@ export const openLink = (url: string, isExternal: boolean = false) => {
   link.click();
 };
 
+const ReferencePathChunks = ['/versions/', '/modules/', '/more/'] as const;
+
+export const isReferencePath = (url: string) => {
+  for (const pathChunk of ReferencePathChunks) {
+    if (url.includes(pathChunk)) {
+      return true;
+    }
+  }
+  return false;
+};
```

Comment:
Can we use array `some` instead of for loop?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Updated the code. Thanks for the tip!